CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 CHANGELOG
 =========
 
-0.11.0 (2019-09-13)
+0.12.0 (2019-10-07)
 -------------------
 
-- Add support of templates when values are dict
+- Add Jinja2 'include' support for template command (#116)
```

Comment:
Hm, my bad, I should have checked and documented how the changelog works. You removed the changelog of the previous version. What you need to do is to add a new section, like:
```
0.12.0 (unreleased)
-------------------

- Add Jinja2 'include' support for template command (#116)
```
Without modfiying the changelog vor version 0.11.0.

Again, sorry, this should really have been documented :o

Reply:
Self-facepalm! Replacing last entry definitely wasn't intentional. Fixed.